cookbooks/ros2_windows/recipes/rti_connext.rb

```diff
@@ -1,17 +1,35 @@
 connext_params = node['ros2_windows']['rti_connext']
 
 # These will fail if the rti_connext parameters have not been specified because the defaults are 'nil'
-assert(File.exists?(connext_params['license_file']),
-                    "License file location does not exist: #{connext_params['license_file']}")
+unless File.exists?(connext_params['license_file']) do
+  Chef::Log.fatal("License file location does not exist: #{connext_params['license_file']}")
+  raise
+end
+
+unless Dir.exists?(connext_params['installer_dir']) do
+  Chef::Log.fatal("Installer directory does not exist: #{connext_params['installer_dir']}")
+  raise
+end
+
+if connext_params['version'] == nil do
```

Comment:
Nitpick. The `.nil?` method is preferred to `== nil`.
```suggestion
if connext_params['version'].nil? do
```

Reply:
Fixed, also changed this from an if do loop (which is _not_ a thing) to an if then loop (which _is_ a thing)